params/config.go

```diff
@@ -97,6 +97,7 @@ var (
 		GingerbreadBlock:    big.NewInt(18785000),
 		GingerbreadP2Block:  big.NewInt(19157000),
 		HForkBlock:          nil, // TBD
+		L2MigrationBlock:    big.NewInt(27109999),
```

Comment:
The L2MigrationBlock is the one that is not going to generate. In this case, if we set the 9999, we will have a gap of 1 block between the L1 and L2
```suggestion
		L2MigrationBlock:    big.NewInt(27110000),
```